src/poprox_storage/repositories/articles.py

```diff
@@ -34,24 +34,36 @@ def __init__(self, connection: Connection):
             "impressions",
         )
 
-    def fetch_todays_articles(self) -> list[Article]:
-        return self.fetch_articles_since(days_ago=1)
-
-    def fetch_past_articles(self) -> list[Article]:
-        return self.fetch_articles_before(days_ago=1)
-
```

Comment:
There's only one usage of either of these methods, which is used in `poprox-platform` to fetch the candidate pool for today's newsletter. Removed in the corresponding PR that uses the new methods below.